recipes-kernel/linux/linux-raspberrypi.inc

```diff
@@ -111,10 +111,6 @@ do_configure_prepend() {
         kernel_configure_variable SQUASHFS y
         kernel_configure_variable UBIFS_FS y
     fi
-    # Activate CONFIG_LEGACY_PTYS
-    kernel_configure_variable LEGACY_PTYS y
-    # this module is built externally via drbd-utils
-    kernel_configure_variable BLK_DEV_DRBD n
```

Comment:
I think disabling this is still the right choice, we should document how to enable drbd from meta-networking instead

Reply:
this lets us use meta-networking out of box, if there are no users of drbd who do not use meta-networking as well then your change makes sense, but thats a smaller set than the other

Reply:
Ok. I'm trying to avoid us carrying any config changes we don't have a good reason for. Supporting the recipes in meta-networking out-of-the-box is probably worthwhile though. I'll add this back in but keep CONFIG_LEGACY_PTYS disabled.

Reply:
Fixed.

Reply:
Fail. drbd in meta-networking doesn't build against Linux v4.14.y. The version in meta-networking is v9.0.8, compatibility with this kernel version was added in v9.0.10 according to drbd release notes (https://github.com/LINBIT/drbd-9.0/blob/master/ChangeLog).

So I think we should go back to leaving BLK_DEV_DRBD enabled.